repo/packages/N/nexus/0/config.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
I kind of doubt that default values make sense in every case, as for the host volume or external persistent volume configuration. At least the hostname to pin the service to needs to be defined, as well as the host volume path needs to exist as a possible Docker volume mount point.

Afaik one cannot automate this. The GitLab package doesn't do this either: https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/G/gitlab/2/config.json#L193